### PR TITLE
Fix invalid Google URLs and invision records

### PR DESCRIPTION
**Pull request by pdehaan**
r+ @k88hudson

### CODE REVIEW

File: top_sites.json [hunk 1]
```diff
@@ -731,7 +731,7 @@
   {
     "title": "Invision",
     "url": "https://www.invisionapp.com/",
-    "image_url": "invisionapp-com.png",
+    "image_url": "invision-com.png",
```
Comment (pdehaan) [inline top_sites.json:726]: was

```
  1) top_sites #104 Invision should have an image_url pointing to a .png or .svg image prefixed with images/:
     AssertionError: expected invisionapp-com.png to be found on disk: expected false to be true
```

File: top_sites.json [hunk 2]
```diff
@@ -530,98 +530,98 @@
   },
   {
     "title": "Google Admin",
-    "url": "http:/admin.google.com/",
+    "url": "http://admin.google.com/",
```
Comment (pdehaan) [inline top_sites.json]: Not sure if these should all be HTTPS, or if we just live with a redirect.

File: top_sites.json [hunk 3]
```diff
@@ -530,98 +530,98 @@
...
   {
     "title": "Calendar",
-    "url": "http:/calendar.google.com/",
+    "url": "http://calendar.google.com/",
     "image_url": "google-calendar.png",
     "background_color": "#FFF",
     "domain": "google.com"
   },
   {
     "title": "Contacts",
```
Comment (pdehaan) [inline top_sites.json:542]: Not sure if "Contacts" is descriptive enough, or if we make these slightly more verbose w/ "Google Contacts" and "Google Calendar" and "Google Docs", etc.

File: top_sites.json [hunk 4]
```diff
@@ -530,98 +530,98 @@
...
+    "url": "http://admin.google.com/",
     "image_url": "google-admin.png",
     "background_color": "#FFF",
     "domain": "google.com"
   },
   {
     "title": "Calendar",
-    "url": "http:/calendar.google.com/",
+    "url": "http://calendar.google.com/",
     "image_url": "google-calendar.png",
```
Comment (pdehaan) [inline top_sites.json:537]: FWIW, these image names seem to break the trend of calendar-google-com.png, but not sure if we care.

File: top_sites.json [hunk 5]
```diff
@@ -740,7 +740,7 @@
     "url": "https://mozilla.invision.com/",
```
Comment (pdehaan) [inline top_sites.json]: > request to https://mozilla.invision.com/ failed, reason: getaddrinfo ENOTFOUND mozilla.invision.com mozilla.invision.com:443

File: top_sites.json (hunk 2 above)
Comment (k88hudson) [reply top_sites.json]: Let's change them to `https://`

File: top_sites.json (hunk 4 above)
Comment (k88hudson) [reply top_sites.json:537]: eh, i'm ok with it. as long as we have tests

File: top_sites.json (hunk 3 above)
Comment (k88hudson) [reply top_sites.json:542]: Looks like `Google Calendar` doesn't quite fit, so maybe that's why they're shorter

File: top_sites.json [hunk 6]
```diff
@@ -731,16 +731,16 @@
...
     "url": "https://www.invisionapp.com/",
-    "image_url": "invisionapp-com.png",
+    "image_url": "invision-com.png",
     "background_color": "#ff2e63",
     "domain": "invisionapp.com"
   },
   {
     "title": "Invision",
-    "url": "https://mozilla.invision.com/",
+    "url": "https://mozilla.invisionapp.com/",
```
Comment (k88hudson) [inline top_sites.json:732]: This is pretty darn specific but ok 😜

**Review by k88hudson — APPROVED** [review]
Looks great! R+ but I would recommend using the https for all the Google stuff.

**Comment by pdehaan** [comment]
Awesome, thanks @k88hudson.
Not sure if we have anything else we want to add to tippy-top-sites, or if we tag and publish a new version to npm now so we can update activity-stream to latest tippy-top for optimal iconery.

**Comment by k88hudson** [comment]
@pdehaan let's tag it and publish! if we need to add more, we can always do another release

**Comment by pdehaan** [comment]
A fun bit of trivia...  Here's the general module file size (via `$ du -sh node_modules/tippy-top-sites`) of the 3 releases of **tippy-top-sites**:

```
0.1.0   1.6M    node_modules/tippy-top-sites
0.2.0   776K    node_modules/tippy-top-sites
0.3.0   868K    node_modules/tippy-top-sites
```

**Pull request merged**